Snakemake/workflow/rules/prepare_files_for_tsinfer.smk

```diff
@@ -84,33 +84,36 @@ rule match_ancestral_vcf:
     # test if the position existis in the ancestral file (2),
     # if TRUE print the line from file (2),
     # else print the line from file (3)
+    # at the end remove POS column
         """
         grep "{wildcards.chromosome}_" {input.ancestral} > aa_tmp
 
         for line in $(cat {input.vcfPos});
         do
-            if grep -w "${line}" aa_tmp >> {output}; then
+            if grep -w "${line}" aa_tmp >> {tmp}; then
                 continue
             else
-                grep -w "${line}" {input.major} >> {output}
+                grep -w "${line}" {input.major} >> {tmp}
             fi
         done
-#        awk -F"," '{{print $1"\t"$2}}' tmp > {output}
+        cut -d',' -f 2 tmp > {output}
```

Comment:
Why are we taking just the second column? The next rule is written in a way that it takes the second column so we can leave this column in (it might be useful if someone wants to inspect it) - so I would leave this last step out